adblock/rules_test.go

```diff
@@ -137,14 +137,17 @@ func TestDomainAnchor(t *testing.T) {
 
 func TestOptsDomain(t *testing.T) {
 	testInputs(t, `
-/ads$domain=foo.com|~info.foo.com
+/ads$domain=foo.com
+/ads$domain=bar.com
+/ads$domain=~info.foo.com
```

Comment:
Splits domain options into multiple rule lines to make sure logic works across rules.